docs/introduction/entity-component-system.md

```diff
@@ -451,27 +451,37 @@ Now we can include it into our HTML:
 </html>
 ```
 
+### Using jsdelivr for CDN
+
+[jsdelivr]: https://www.jsdelivr.com/
+
+[JSDELIVR][jsdelivr] is an alternative CDN to unpkg.  One benefit of JSDELIVR is that it can download files from GitHub as well as NPM.
+
+You can convert unpkg URLs to  JSDELIVR URLs using this link: https://www.jsdelivr.com/unpkg
+
+You can convert GitHub URLs to  JSDELIVR URLs using this link: https://www.jsdelivr.com/github
+
 ### Example
 
-[glitch]: http://glitch.com/~aframe-registry
+[community-example]: https://aframe.io/examples/showcase/community-components
 
-![Registry Example](https://cloud.githubusercontent.com/assets/674727/25502318/0f76ceec-2b4b-11e7-9829-cb3784b20dc1.gif)
+![Community Components Example](https://cloud.githubusercontent.com/assets/674727/25502318/0f76ceec-2b4b-11e7-9829-cb3784b20dc1.gif)
 
 Below is a complete example of using various community components from the
-Registry and using the unpkg CDN. We can [remix or check out this example on
-Glitch][glitch].
+Registry and using the JSDELIVR CDN.  This example can also be viewed in the [A-Frame Examples][community-example].
 
 ```html
 <html>
   <head>
-    <script src="https://aframe.io/releases/1.4.0/aframe.min.js"></script>
-    <script src="https://unpkg.com/aframe-animation-component@3.2.1/dist/aframe-animation-component.min.js"></script>
-    <script src="https://unpkg.com/aframe-particle-system-component@1.0.x/dist/aframe-particle-system-component.min.js"></script>
-    <script src="https://unpkg.com/aframe-extras.ocean@%5E3.5.x/dist/aframe-extras.ocean.min.js"></script>
-    <script src="https://unpkg.com/aframe-gradient-sky@1.4.0/dist/gradientsky.min.js"></script>
+    <title>Community Components Example</title>
+    <meta name="description" content="Community Components Example">
+    <script src="../../../dist/aframe-master.js"></script>    
```

Comment:
For embedded code in docs we can point to the A-Frame version. That gets updated automatically. If we point to dist/aframe-master once cannot copy / paste the code.  

One of the features I once requested to glitch is having a glitch backed up by a git directory. These way we could have all the glitches in a single repo and update them all on a new release.

Reply:
FYI for networked-aframe, I update the naf-examples glitch by opening the glitch terminal and git pull from github, see my procedure https://github.com/networked-aframe/networked-aframe/blob/master/docs/release-instructions.md
Each glitch is actually a git repository where it autocommit the changes.

Reply:
I'm proposing these lines in each example:

```
    <!-- If running this example without a local copy of A-Frame, replace this next line with:
    <script src="https://aframe.io/releases/<release_number>/aframe.min.js"></script>
    -->
    <script src="../../../dist/aframe-master.js"></script>   
```

Not perfect, but feels like a reasonable compromise?

Same applies to #5213, #5214, #5215.

Reply:
I think this embedded code should point to the latest A-Frame version [as it was before](https://github.com/aframevr/aframe/pull/5211/files#diff-91dd1cf1fe2f714a59bcfe0a8260e0d2b28b1a4363e224280ce654124d2bf6a3L467). The doc update script updates the version automatically upon publication. Does it make sense?

Reply:
OK, makes sense.  Changed back to 1.4.0 version.  Quoted code in the docs won't exactly match the code in the linked example, but the difference is hopefully not going to be too confusing to anyone...